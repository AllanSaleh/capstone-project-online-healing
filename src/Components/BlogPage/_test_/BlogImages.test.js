import * as React from 'react';
import BlogImages from "../BlogImage";
import * as renderer from "react-test-renderer";

it('Blog Images renders correctly', () => {
  const tree = renderer
    .create(<BlogImages/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});