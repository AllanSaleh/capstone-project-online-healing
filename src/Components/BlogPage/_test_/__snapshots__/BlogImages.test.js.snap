// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Images renders correctly 1`] = `
<div
  className="flex items-center lg:h-firstsection px-sides pt-24 lg:pt-navbar"
>
  <img
    alt="Blog"
    className="w-full"
  />
</div>
`;
